packages/react-codemirror/package.json

```diff
@@ -18,22 +18,15 @@
     "codemirror 6"
   ],
   "version": "2.0.0-next.4",
-  "main": "./dist/cjs/index.cjs",
-  "module": "./dist/esm/index.mjs",
+  "main": "./dist/esm/index.mjs",
   "types": "./dist/types/index.d.ts",
-  "exports": {
-    ".": {
-      "require": "./dist/cjs/index.cjs",
-      "import": "./dist/esm/index.mjs",
-      "default": "./dist/cjs/index.cjs"
-    }
-  },
+  "type": "module",
   "sideEffects": false,
   "scripts": {
-    "build": "concurrently 'npm:build-types' 'npm:build-esm' 'npm:build-commonjs'",
+    "build": "concurrently 'npm:build-types' 'npm:build-esm' 'npm:build-worker'",
     "build-types": "tsc --emitDeclarationOnly --outDir dist/types",
     "build-esm": "esbuild ./src/index.ts --bundle --format=esm --sourcemap --packages=external --outfile=dist/esm/index.mjs",
-    "build-commonjs": "esbuild ./src/index.ts --bundle --format=cjs --sourcemap --packages=external --outfile=dist/cjs/index.cjs",
+    "build-worker": "esbuild ./src/lang-cypher/semantic-analysis-worker.ts --bundle --format=esm --sourcemap --packages=external --outfile=dist/esm/semantic-analysis-worker.mjs",
```

Comment:
note: i removed the cjs build here to get the webworkers to work in a convenient way for the consumer of the package